geoportal/vars.yaml

```diff
@@ -918,6 +920,7 @@ vars:
         - http://localhost:3000
         - http://localhost:3001
         - http://localhost:3002
```

Comment:
From me, we can remove this line :-)

Reply:
Currently, this PR is a draft and don't work, but yes, if we want to use https, we can then remove this line

Reply:
done